src/experimental/patterns/ec2-app.ts

```diff
@@ -41,17 +42,17 @@ export const RollingUpdateDurations: AutoScalingRollingUpdateDurations = {
  *
  * TODO Expose the healthcheck grace period as a property on {@link GuEc2App} and remove this `Aspect`.
  */
-class AutoScalingRollingUpdateTimeout implements IAspect {
+export class AutoScalingRollingUpdateTimeoutExperimental implements IAspect {
```

Comment:
I think this, and other exported classes would need to be [prefixed `Gu` to be picked up by the metadata aspect](https://github.com/guardian/cdk/blob/e416a34f2dbe454169e8351622640a5074286c11/src/aspects/metadata.ts#L27).